focused_proof.ml

```diff
@@ -296,6 +295,13 @@ let rec prove sequent select_d2 max_d2 ttl =
               None
       | _ -> None
 
+let rec prove_with_increasing_bound focused_sequent exponential_bound ttl =
+    match prove focused_sequent [] exponential_bound ttl with
```

Comment:
```suggestion
    has_reached_exponential_bound := false;
    match prove focused_sequent [] exponential_bound ttl with
```

Reply:
`has_reached_exponential_bound` should be reset at each increment of `exponential_bound`

Reply:
Of course 👍🏻